docs/whats-new/dotnet-docs-2020-07.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
Rename to just `2020-07.md`?

Based on @mairaw comment (https://github.com/dotnet/docs/pull/19257/#discussion_r447305779).